inc/renderer.php

```diff
@@ -133,101 +137,105 @@ public function generate_css( $post_id, $panels_data = false, $layout_data = fal
 				! empty( $row['style']['bottom_margin'] ) ||
 				! empty( $panels_margin_bottom_last_row )
 			) {
-				// Filter the bottom margin for this row with the arguments
 				$css->add_row_css( $post_id, $ri, '', array(
-					'margin-bottom' => apply_filters( 'siteorigin_panels_css_row_margin_bottom', $panels_margin_bottom . 'px', $row, $ri, $panels_data, $post_id )
+					'margin-bottom' => $panels_margin_bottom
 				) );
 			}
 
 			$collapse_order = ! empty( $row['style']['collapse_order'] ) ? $row['style']['collapse_order'] : ( ! is_rtl() ? 'left-top' : 'right-top' );
 
-			if ( $settings['responsive'] ) {
-
+			if ( $settings['responsive'] && ! empty( empty( $row['style']['collapse_behaviour'] ) ) ) {
```

Comment:
This is pretty hard to read and seems unnecessary. `! empty( empty( $var ) )` should be the same as just `empty( $var )`?

Reply:
Aaah yeah, that's a copy paste typo. Will fix it up.